README.md

```diff
@@ -13,6 +13,8 @@ set which resolver mode you want.
 
 3 - **Only**. Only use TRR. Never use the native (after the initial setup).
 
+  - **Note:** `network.trr.bootstrapAddress` must be set for resolution to work in TRR mode 3. See https://bugzilla.mozilla.org/show_bug.cgi?id=1600976
+
```

Comment:
Maybe it should say something that it is needed *if the given URL uses a host name* or something? `https://1.1.1. 1` for example shouldn't need any bootstrap...

Reply:
done